src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +39,9 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
 
     @Override
-    Player getUser();
-
+    User getUser();
```

Comment:
`Player` extends `User`. Why would we not override to return `Player` in a `PlayerEvent`? There is no other "object" that represents the `User` that is a `Player` since the `Player` IS the `User`.